src/hb-set.cc

```diff
@@ -61,6 +61,22 @@ hb_set_create ()
   return set;
 }
 
+/**
+ * Creates a set initially populated with num_codepoint codepoints, which must
+ * be in increasing order.
+ *
+ * Since: REPLACEME
+ */
+HB_EXTERN void
+hb_set_add_sorted_array (hb_set_t             *set,
+		         const hb_codepoint_t *sorted_codepoints,
+		         unsigned int          num_codepoints)
+{
+  set->add_sorted_array (sorted_codepoints,
+		         num_codepoints,
+		         sizeof(hb_codepoint_t));
+}
+
```

Comment:
This should probably go after `hb_set_add` below. The documentation syntax needs to be fixed as well, and the new method need to be added to the relevant section in `docs/harfbuzz-sections.txt`.